scripts/process_guard.py

```diff
@@ -96,47 +97,45 @@ def pid_exists(pid):
 
     def get_raw_stats(self):
         for pid in self.pid_list:
-            try:
-                if False:
-                    status = "9932 (bash) S 1 9932 9932 0 -1 8192 1330 25068 0 12 1 0 21 7 20 0 1 0 873934607 112930816 2 18446744073709551615 1 1 0 0 0 0 65536 4 65538 18446744073709551615 0 0 17 0 0 0 0 0 0"
-                    lines = [
-                        "rchar: 2012\n",
-                        "wchar: 0\n",
-                        "syscr: 7\n",
-                        "syscw: 0\n",
-                        "read_bytes: 0\n",
-                        "write_bytes: 0\n",
-                        "cancelled_write_bytes: 0\n",
-                    ]
-                else:
+            if not os.path.exists('/proc'):
+                dummy_status = "%s (sh) S 0 0 0 0 -1 8192 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 " \
```

Comment:
What is the total expected size of records here?

Reply:
52 items, see the procfs manual: http://man7.org/linux/man-pages/man5/proc.5.html